### PR TITLE
Merge Developers into Build page 

### PR DESCRIPTION
**Pull request by michellewong793**
Hello! There are a few changes here. 

In Contentful, /build is switched to /developers. We may have to set up redirects? 

Next, the FullStack component is spliced into the array of items pulled in front Contentful, formerly known as the Build page. 

screencapture-localhost-3000-developers-2021-08-05-14_36_19

### CODE REVIEW

File: src/dev/DevelopersPage.tsx [hunk 1]
```diff
@@ -1,40 +1,21 @@
...
-    return (
-      <View style={styles.container}>
-        <OpenGraph
-          image={previewImage}
-          path={menuItems.BUILD.link}
-          title={"Celo Developers | Celo SDK"}
-          description={
-            "Innovative tools to build native mobile dApps. Celo is a proof-of-stake based blockchain with smart contracts that allows for an ecosystem of powerful applications."
-          }
-        />
```
Comment (aaronmgdr) [inline src/dev/DevelopersPage.tsx:25]: we still need an OpenGraph 

```
<OpenGraph
        image={props.openGraph?.fields?.file?.url}
        title={props.title}
        description={props.description}
        path={props.slug}
      />
```

Comment (michellewong793) [reply src/dev/DevelopersPage.tsx:25]: ty!

File: src/_page-tests/developers/Index.test.tsx [hunk 2]
```diff
@@ -2,13 +2,73 @@ import * as React from "react"
...
+import { BLOCKS } from "@contentful/rich-text-types"
+
+function blurbFactory(unique): Entry<BlurbProps> {
+  return {
+    sys: {
+      id: unique,
+      type: "",
+      locale: "en",
+      createdAt: "2021-07-01",
+      updatedAt: "2021-07-01",
```
Comment (michellewong793) [inline src/_page-tests/developers/Index.test.tsx:17]: this test data is from the HomePage tests

File: server/index.ts [hunk 3]
```diff
@@ -163,23 +163,42 @@ function wwwRedirect(req: express.Request, res: express.Response, nextAction: ()
...
           return
         }
-        const userId = req.params.userId ?? ""
-        if (!/\w+/.test(userId)) {
-          res.sendStatus(400)
-          return
-        }
         res.sendFile(
-          path.join(host, "hu", userId),
+          path.join(host, "policy"),
```
Comment (aaronmgdr) [inline server/index.ts:201]: what is this?

Comment (aaronmgdr) [reply server/index.ts:201]: oh i see its not the change it appears to be

**Comment by aaronmgdr** [comment]
code seems good.
can you add some screenshots to the PR? 

and see what is up with that test

File: server/index.ts (hunk 3 above)
Comment (michellewong793) [reply server/index.ts:201]: yeah idk, prettier did some things, all i did was add "/build" to the redirect and theres some changes to spacing

**Comment by michellewong793** [comment]
> code seems good.
> can you add some screenshots to the PR?
> 
> and see what is up with that test

yep checking it out

**Review by aaronmgdr — APPROVED** [review]
this seems good

**Comment by aaronmgdr** [comment]
🚀🎉

**Pull request merged**